### DISCUSSION TITLE
Blogs 5.2.0

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#4558] [General]
The open source Oqtane.Blogs module (https://github.com/oqtane/oqtane.blogs) had a significant release today... version 5.2.0 is now available for installation via the Marketplace.

This version has the following enhancements:

- content versioning support
- draft / published workflow
- ability to include a main blog thumbnail image
- category support
- tag support
- user comment support
- content indexing support (via the new site search)
- advanced SEO support
- ability to modify the Url slug
- ability to specify Publish Date
- email subscription support including RSS feed
- blog widgets for displaying different views of blogs (ie. most recent, most popular, related posts, etc...)
- templating via tokenization

The module can be used for standard blogging - or can be used for publishing articles. 

Blogs 5.2.0 is running on https://www.oqtane.org/blog

**Comment by ADefWebserver** [comment]
@sbwalker - Thank you for doing this!

**Reply by sbwalker** [reply]
@ADefWebserver @zyhfish did most of the heavy lifting on these enhancements... I just did some minor modifications based on testing/upgrade.

**Comment by sbwalker** [comment]
Add Blog:

![image](https://github.com/user-attachments/assets/970c75c4-b429-4278-b589-13f3dc58d517)

Blog Settings:

![image](https://github.com/user-attachments/assets/fc68dc79-1c2f-4034-850f-b1bda0bd0c78)
